docs/releases/2.2.13.txt

```diff
@@ -9,4 +9,6 @@ Django 2.2.13 fixes two security issues and a bug in 2.2.12.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression in Django 2.2.12 that affected translation loading for
```

Comment:
`Django 2.2.13 fixes two security issues and a bug in 2.2.12.` ->
`Django 2.2.13 fixes two security issues and a regression in 2.2.12.`